easybuild/easyconfigs/e/ELSI/ELSI-2.5.0-foss-2019b_slepc.eb

```diff
@@ -0,0 +1,35 @@
+name = 'ELSI'
+version = '2.5.0'
+versionsuffix = '_slepc'
```

Comment:
@migueldiascosta I would use `-SLEPc` instead?

How exactly is an `ELSI` installation that depends on `SLEPc` different? It's not just an enhanced installation compared to without `SLEPc` dep?

Reply:
There's a dependency conflict between the internal `PEXSI` and `SLEPc`, hence the variants. We could make the one with `SLEPc` the default, though...